src/descriptors/power_system_structs.json

```diff
@@ -1245,6 +1245,159 @@
             ],
             "supertype": "DCBranch"
         },
+        {
```

Comment:
What's the difference between this and a T line without the shunt? For PSI the shunt doesn't really matter.